### ISSUE TITLE
Mathjax error loading TeX font

### ISSUE THREAD

**Issue by wyschean** [ocaml-sf/learn-ocaml#119]
Currently ``static/js/mathjax/fonts/HTML-CSS/TeX/*`` doesn't exist, so mathjax processing stalls until it finds a fallback font

**Comment by AltGr** [comment]
Maybe the paths are not well configured ? `learn-ocaml` normally puts the fonts at `<webroot>/js/mathjax/jax/output/HTML-CSS/fonts/TeX`. I'll check.

**Comment by wyschean** [comment]
That's the location of `.js` files mathjax uses to render font data. The actual font data (the `.woff` files) should live in `<webroot>/js/mathjax/fonts/HTML-CSS/TeX/woff`.

**Comment by AltGr** [comment]
Ah, ok, thanks for the clarification. Looking into it.

**Comment by wyschean** [comment]
We have also encountered some instances (on macOS: one on Safari 11.1.2, another on Chrome 69.0.3497.81, and some more on other macOS machines) where mathjax was attempting to load the STIX fonts that were removed from `learn-ocaml`'s distribution, resulting in a `[Math Processing Error]`. There's probably some weird interaction with mathjax on macOS browsers.

If needed I can write a PR to hard-code STIX out of the mathjax config to see if that helps.
For now, our workaround is to just copy the font files from the mathjax repo into our webroot.
